src/components/main/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import styles from '../../css-modules/main.module.css';
+
+// Utils
+import sortedBy from '../../utils/sortedBy';
+import camelCase from '../../utils/camelCase';
+
+// Components
+import NavBar from './navbar';
+import ListMovies from './ListMovies';
+
+const movies = [
+    { id: 1, title: "Pulp Fiction", releaseDate: "2004", genres: [ "Action", "Adventure" ], image: "/img/pulpFiction.png", rating: 7.8, movieUrl: "", runtime: 0, overview: "" },
+    { id: 2, title: "Bohemian Rhapsody", releaseDate: "2003", genres: [ "Drama", "Biography", "Music" ], image: "/img/bohemianRhapsody.png", rating: 8, movieUrl: "", runtime: 10, overview: "" },
+    { id: 3, title: "Kill Bill: Vol 2", releaseDate: "1994", genres: [ "Oscar Winning Movie" ], image: "/img/killBill.png", rating: 9, movieUrl: "", runtime: 50, overview: "" },
+    { id: 4, title: "Avengers: Infinity War", releaseDate: "2018", genres: [ "Action", "Adventure" ], image: "/img/avengers.png", rating: 7, movieUrl: "", runtime: 180, overview: "" },
+    { id: 5, title: "Inception", releaseDate: "2010", genres: [ "Action", "Adventure" ], image: "/img/inception.png", rating: 6, movieUrl: "", runtime: 100, overview: "" },
+    { id: 6, title: "Reservoir Dogs", releaseDate: "1997", genres: [ "Oscar Winning Movie" ], image: "/img/reservoirDogs.png", rating: 5, movieUrl: "", runtime: 120, overview: "" },
+]
+
+const Main = () => {
+
+    // Local State
+    const [ typeSelected, setTypeSelected ] = useState('all');
+    const [ sortBySelected, setSortBySelected ] = useState('date');
+    const [ moviesSelected, setMoviesSelected ] = useState([]);
```

Comment:
do you really need to put these movies in the state?

Reply:
I made this to sort the movies list

Reply:
But you don't really need to store it in the state, you can just sort in place